templates/dialogs/fight.hbs

```diff
@@ -21,17 +21,55 @@
             <label for="showFightV3">Show Volley 3</label>
         </div>
         {{else}}
-        <div>Fighters</div><div>Volley 1</div><div>Volley 2</div><div>Volley 3</div>
+        <div class="fighters-label">Fighters</div><div>Volley 1</div><div>Volley 2</div><div>Volley 3</div>
         {{/if}}
         {{#each participants as |p i|}}
         <div class="volley-buttons flex-row">
             <div class="participant">{{p.name}}</div>
-            {{#if p.showActions}}
-            <button class="control-button" data-index="{{i}}" data-action="rollSpeed">Test Speed</button>
-            <button class="control-button" data-index="{{i}}" data-action="rollPower">Test Power</button>
-            <button class="control-button" data-index="{{i}}" data-action="rollAgility">Test Agility</button>
-            <button class="control-button" data-index="{{i}}" data-action="rollSkill">Test Skill</button>
-            {{/if}}
+            <div class="portrait-actions">
+                {{#if p.showActions}}
+                <img class="portrait clickable" src="{{p.imgSrc}}" title="{{p.name}}" data-action="openSheet" data-actor-id="{{p.id}}"/>
+                <div class="fight-rolls">
+                    <button class="control-button" data-index="{{i}}" data-action="rollSpeed">Speed</button>
+                    <button class="control-button" data-index="{{i}}" data-action="rollPower">Power</button>
+                    <button class="control-button" data-index="{{i}}" data-action="rollAgility">Agility</button>
+                    <button class="control-button" data-index="{{i}}" data-action="rollSkill">Skill</button>
+                    <button class="control-button" data-index="{{i}}" data-action="rollSteel">Steel</button>
+                </div>
+                {{else}}
+                    {{#if ../gmView}}
+                        <div class="details clickable" data-action="openSheet" title="{{p.name}}" data-actor-id="{{p.id}}">
+                        <table>
```

Comment:
Hard 👎 on this design. You might note that there are exactly no tables being used in burning wheel. If you want to organize things into a grid, use a div with `display: grid`.